virttest/env_process.py

```diff
@@ -526,7 +527,11 @@ def validate_memory_resource():
         if (params.get("setup_ksm") == "yes" and
                 params.get("ksm_run", "1") == "1"):
             magnification = 1.2
-        free_mem = "%s KB" % memory.freememtotal()
+        # Test whether is MemAvailable already introduced
+        if 'MemAvailable' in genio.read_file('/proc/meminfo'):
+            free_mem = "%s KB" % memory.read_from_meminfo('MemAvailable')
```

Comment:
How about append the memory module in avocado, which already has 'freememtotal()' for 'MemFree', so we can call it directly without import a new module(genio)